docker-deploy-prod.sh

```diff
@@ -39,7 +39,7 @@ then
       service="my-dev-space-users-prod-service"
       template="ecs_users_prod_taskdefinition.json"
       task_template=$(cat "ecs/$template")
-      task_def=$(printf "$task_template" $AWS_ACCOUNT_ID $AWS_RDS_URI $PRODUCTION_SECRET_KEY)
+      task_def=$(printf "$task_template" $AWS_ACCOUNT_ID $AWS_RDS_USERS_URI $SECRET_KEY $NEW_RELIC_LICENSE_KEY)
```

Comment:
Injects the following environment variables into the task definitions using the natural order of evaluation (first to last) in the corresponding `ecs/ecs_*_prod_taskdefinition.json` files